### DISCUSSION TITLE
Is there any chance you will update

### DISCUSSION THREAD

**Discussion by KjeldsenDK** [vladmandic/pigallery#19] [Q&A] [answered]
This program and the instructions? I have been looking all over the internet and afaik there are few if any programs that does the same as this does. I would love to run this on my PI4 and use it when i need to find "things" in my photo collection.

**Comment by vladmandic** [comment, marked as answer]
its been long overdue, i'll see if i can pick it up again and modernize a bit, its been a long time since the last update.

**Reply by KjeldsenDK** [reply]
Thank you for answering.
As I said, from my research your program is pretty unique. And considering whats been happening the past years it was ahead of it's time.
